modules/bringauto_context/ContextManager.go

```
@@ -150,6 +197,68 @@ func (context *ContextManager) GetPackageWithDepsJsonDefPaths(packageName string
+				if recursively {
+					packsDepsOnRecursive, err := context.GetDepsOnJsonDefPaths(config.Package.Name, false)
+					if err != nil {
+						return []string{}, err
+					}
+					packsToBuild = append(packsToBuild, packsDepsOnRecursive...)
+				}
+				break
+			}
+		}
+	}
+	packsToRemove, err := context.GetPackageWithDepsJsonDefPaths(packageName)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)